lib/utils/display.js

```diff
@@ -239,7 +239,7 @@ class Display {
     this.#command = command
     this.#levelIndex = LEVEL_OPTIONS[loglevel].index
     this.#timing = timing
-    this.#json = json
+    this.#json = ['pkg'].includes(command) ? true : json
```

Comment:
This is probably a config that the pkg command should set.  See `ll` `link` and `find-dupes` for examples of this.

Reply:
I updated the PR to do it that way not. It does require passing json to `ouput.flush` because the json option has changed between config loading and output ending. It's a little more code but I think makes it clearer what's happening when looking at the `Pkg` command.